dataland-frontend/src/components/resources/frameworkDataSearch/ThreeLayerDataTable.vue

```diff
@@ -188,25 +193,37 @@ export default defineComponent({
     /**
      * Iterates through all subcategories of a category
      * @param categoryDataObject the data object of the framework's category
-     * @param categoryKey the key of the corresponding framework's category
      * @param category  the category object of the framework's category
      * @param dataId  the ID of the dataset
      * @param listOfKpiDataObjects collector for the kpi data objects
      * @param currentViewModelDataset dataset for which the show if conditions should be checked
      */
     iterateThroughSubcategories(
       categoryDataObject: object,
-      categoryKey: string,
       category: Category,
       dataId: string,
       listOfKpiDataObjects: Array<KpiDataObject>,
       currentViewModelDataset: FrameworkViewModel,
     ) {
       for (const [subCategoryKey, subCategoryObject] of Object.entries(categoryDataObject)) {
-        if (subCategoryObject == null) continue;
+        let adaptedSubCategoryObject = subCategoryObject as ObjectType;
+        if (subCategoryObject == null) {
+          continue;
+        }
+        if (subCategoryKey == "totalAmount") {
```

Comment:
I am not sure why this is happening. The CurrencyDataPoint is recast into an ExtendedDataPoint with a monetray amount as T.

Reply:
UI requested that on the view page the corresponding field of totalAmount should contain value and currency in one field 
![image](https://github.com/d-fine/Dataland/assets/137428621/3b8d927c-c4e9-402c-87d4-c0a616135d53)

we can either convert it to a extendedDataPoint or add a case distinction in the function formatValueForDisplay in the euTaxoNonFinancialsPanel.vue 
We can have a talk about what is preferable here